### ISSUE TITLE
Conditional Card not supporting gauge card

### ISSUE THREAD

**Issue by PierreScerri** [home-assistant/home-assistant-polymer#2083]
**Home Assistant release with the issue:** 0.82.1

**Last working Home Assistant release (if known):**

**Browser and Operating System:** Safari - MacOS Mojave

**Description of problem:**
sensor.energy_flow state is either 'Import' or ' Export'
I wish to display one gauge or the other depending on this state.
Nothing is displayed.
Changing the card type to 'entities' instead of 'gauge' makes it work.

 ```
     - type: conditional
        conditions:
          - entity: sensor.energy_flow
            state: "Import"
        card:
          type: gauge
          title: Import
          entity: sensor.grid_power
      - type: conditional
        conditions:
          - entity: sensor.energy_flow
            state: "Export"
        card:
          type: gauge
          title: Export
          entity: sensor.grid_power
```

**Javascript errors shown in the web inspector (if applicable):** None

**Additional information:** sensor.grid_power has +ve values for Import and -ve values for Export.

Tried using one gauge but does not display meaningful info.

**Comment by zsarnett** [comment]
Can you get a gauge card to work with out the conditional?

**Comment by PierreScerri** [comment]
Yes

**Comment by PierreScerri** [comment]
Yes

**Comment by zsarnett** [comment]
This is caused by the client width initially being 0. We are working on a fix for this

**Comment by PierreScerri** [comment]
Thank you for the info.

At least I know I'm not going crazy.

**Comment by VDRainer** [comment]
Putting the gauges in vertical stacks, shows the expected one after LL refresh.
After the condition changes, the first gauge gets hidden and the second needs a LL refresh to show.
```
     - type: conditional
        conditions:
          - entity: binary_sensor.e3dc_battery_power_pos
            state: 'on'
        card:
          type: vertical-stack
          cards:
            - type: gauge
              entity: sensor.e3dc_battery_power

      - type: conditional
        conditions:
          - entity: binary_sensor.e3dc_battery_power_pos
            state: 'off'
        card:
          type: vertical-stack
          cards:
            - type: gauge
              entity: sensor.e3dc_battery_power_pos
```
I think this is a very useful function and should be fixed.

**Comment by iantrich** [comment]
This looks to have been fixed with #2456. Closing, let me know if not fixed for you.
